### ISSUE TITLE
Broken link

### ISSUE THREAD

**Issue by j-dc** [MicrosoftDocs/windows-itpro-docs#8502]
Broken link:

> **Applies to:**
>      [Microsoft Defender Advanced Threat Protection (Microsoft Defender ATP)](https://go.microsoft.com/fwlink/p/?linkid=2069559)
> 

=> this link (https://go.microsoft.com/fwlink/p/?linkid=2069559)  does not work

it redirects to:  https://query.prod.cms.rt.microsoft.com/cms/api/am/binary/RE2PBG7

and that page has this content:

```
{
    Code: "DocumentGone",
    Message: "StorageDocumentNotFoundException : Document with id 'RE2PBG7', tenant 'am' is now gone (i.e. the document probably expired and thus no longer available).",
    Resource: null
}
```

---
#### Documentdetails

⚠ *Dit gedeelte niet bewerken. Het is vereist om problemen te koppelen tussen docs.microsoft.com en GitHub.*

* ID: cb904674-d586-0a38-0e7c-7ac3e06cbcd9
* Version Independent ID: 549fbb52-6234-2a79-8604-85498e622f98
* Content: [Enable hardware-based isolation for Microsoft Edge (Windows 10) - Windows security](https://docs.microsoft.com/nl-be/windows/security/threat-protection/microsoft-defender-application-guard/install-md-app-guard#install-application-guard)
* Content Source: [windows/security/threat-protection/microsoft-defender-application-guard/install-md-app-guard.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/threat-protection/microsoft-defender-application-guard/install-md-app-guard.md)
* Product: **w10**
* GitHub Login: @denisebmsft
* Microsoft Alias: **deniseb**

**Comment by RAJU2529** [comment]
@j-dc . Thanks for reporting. I am fixing the broken link now.

**Comment by illfated** [comment]
Should be OK to close this ticket now.

1. URL corrected in PR #8516 , commit https://github.com/MicrosoftDocs/windows-itpro-docs/commit/50d30680988504559983d915fc98464f020d90b3
2. Link repaired in commit https://github.com/MicrosoftDocs/windows-itpro-docs/commit/049d42c6f904d75e4ad2e760eb7a3db81c3b133a **pencil edit** (adding back incorrectly removed parenthesis)

**Comment by denisebmsft** [comment]
Thank you @illfated
